src/Sulu/Component/Content/Tests/Unit/Rlp/Strategy/RlpStrategyTest.php

```diff
@@ -171,15 +171,12 @@ protected function tearDown()
 
     public function testIsValid()
     {
-        // false from mapper (is not unique)
         $result = $this->strategy->isValid('/products/machines', 'default', 'de');
-        $this->assertFalse($result);
+        $this->assertTrue($result);
```

Comment:
Ok, because you've adapted the tests... Really can't think that this is a good idea...